torchbenchmark/__init__.py

```diff
@@ -25,11 +25,16 @@ def _test_https(test_url='https://github.com', timeout=0.5):
 
 def _install_deps(model_path):
     if os.path.exists(os.path.join(model_path, install_file)):
-        subprocess.check_call([sys.executable, install_file], cwd=model_path)
+        try:
+            subprocess.check_call([sys.executable, install_file], cwd=model_path)
+        except subprocess.CalledProcessError:
+            print(f"Error while running {model_path}/{install_file}")
+            sys.exit(-1)
```

Comment:
in your comments i thought you said your changes allowed the install to complete even if some models didn't work in py3.8.  this looks like it just bails and later models wouldn't be attempted?

Reply:
The old version would also bail (by raising an exception), but it was impossible to tell which model failed to install.  This just prints out the name of the model causing the failure.